doc/modules/impute.rst

```diff
@@ -190,7 +190,7 @@ Nearest neighbors imputation
 ============================
 
 The :class:`KNNImputer` class provides imputation for filling in missing values
-using the k-Nearest Neighbors approach. By default, a euclidean distance metric
+using the k-Nearest Neighbors approach. By default, a Euclidean distance metric
```

Comment:
I am not sure whether euclidean vs Euclidean is preferred and to be honest, I am not sure this is worth changing.

Maybe revert all this euclidean -> Euclidean changes?